sre/storage/kvStorage_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPutandGet(t *testing.T) {
+
+	m := Map{}
+	tables := []struct {
+		x string
+		y interface{}
+	}{
+		{"a", 1},
+		{"b", 2.0},
+		{"c", "harry"},
+	}
+	for _, table := range tables {
+		m.Put(table.x, table.y)
+		val, _ := m.Get(table.x)
+		// assert.EqualError(t, err, nil)
+		assert.Equal(t, val, table.y)
+	}
+
+}
+func TestDelete(t *testing.T) {
```

Comment:
There's a better way to structure tests.

Try something like this:


```
func TestDelete(t *testing.T) {

    t.Run("Should delete existing key", func(t *testing.T) {
    });
   
    t.Run("Should return an error on non-existing key", func(t *testing.T){});

}
```
Please check for syntactical errors while you code the above.